src/core/Akka.MultiNodeTestRunner.Shared/Persistence/VisualizerRuntimeTemplate.Tree.cs

```diff
@@ -77,31 +77,41 @@ public string BuildOptions(FactData spec)
                                     nodeMessage.TimeStamp))
                     .ToList();
 
-            var firstEventTimeStamp = events.Aggregate(
-                (aggregate, nextValue) =>
-                    aggregate > nextValue
-                        ? nextValue
-                        : aggregate);
+            var startEventTimeParameter = "null";
+            var endEventTimeParameter = "null";
+
+            if (events.Any())
```

Comment:
`.Any()` always allocates an enumerator, when you're just checking for "not empty" then `events.Count > 0` works better.